moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -389,6 +390,27 @@ def clear_path_constraints(self):
         """ Specify that no path constraints are to be used during motion planning """
         self._g.clear_path_constraints()
 
+    def get_trajectory_constraints(self):
+        """ Get the actual trajectory constraints in form of a moveit_msgs.msgs.Constraints """
+        c = Constraints()
+        c_str = self._g.get_trajectory_constraints()
+        conversions.msg_from_string(c, c_str)
+        return c
+
+    def set_trajectory_constraints(self, value):
+        """ Specify the trajectory constraints to be used (as read from the database) """
+        if value == None:
+            self.clear_path_constraints()
```

Comment:
Why this relays to path constraints, while the function is about trajectory constraints?

Reply:
It's a ctrl+c&ctrl+v leftover, fixed now